lib/editor/util/timetable.js

```diff
@@ -12,7 +12,7 @@ import { EXCEPTION_EXEMPLARS } from '.'
 export const SHORTCUTS = {
   offset: ['o', 'SHIFT:+:o', 'i', 'SHIFT:+:i', '-', 'SHIFT:+:-', '+', 'SHIFT:+:+'],
   navigate: ['k:/:j', '←:/:→', 'x', 'a', 'd'],
-  modify: ['#', 'n', 'c', 'SHIFT:+:\'', 'SHIFT:+:;']
+  modify: ['#', 'n', 'c', 'SHIFT:+:\'', 'SHIFT:+:l', 'SHIFT:+:;']
```

Comment:
Is that me, or was copy-from-left SHIFT+Apostrophy? (SHIFT+semicolon does nothing.)